tests/components/prometheus/test_init.py

```diff
@@ -87,6 +87,71 @@
 PROMETHEUS_PATH = "homeassistant.components.prometheus"
 
 
+@dataclass(frozen=True)
+class EntityTestMetricInfo:
+    """Class for all entity data in a single prometheus metric."""
+
+    metric_name: str
+    domain: str
+    friendly_name: str
+    object_id: str
+    metric_value: Any | None = None
```

Comment:
Note that metric values in Prometheus are always floats.

Reply:
Rewritten. That's a good point.